prov/efa/test/efa_unit_tests.c

```diff
@@ -19,6 +19,10 @@ int main(void)
 		cmocka_unit_test(test_rdm_ep_progress_failed_poll),
 		cmocka_unit_test(test_rdm_ep_progress_bad_send_wc_status),
 		cmocka_unit_test(test_rdm_ep_progress_bad_recv_wc_status),
+#ifdef SUPPORT_EX_CQ
+		cmocka_unit_test(test_rdm_ep_progress_recover_forgotten_peer_ah),
```

Comment:
I think in this case the testing target should not be the progress engine, but the new function that we are going to introduce that handle address.